torch/_meta_registrations.py

```diff
@@ -104,6 +104,16 @@ def meta_index_select_out(self, dim, index, out):
 def meta_max(self):
     return self.new_empty(())
 
+@register_meta([aten.max.dim], register_dispatcher=False)
```

Comment:
@ezyang whats the  `register_dispatcher=False` for?  In cases where we already have a functioning dispatcher path for real kernels, we don't want to duplicate registration and the way we hit these is via pythonkey behavior instead? or something

Reply:
This is for when we have a C++ meta kernel. We don't register these to the dispatcher, but we do put them in a Python table so we can use them for symbolic shapes, since the C++ meta functions usually do not support symbolic shapes. There are a few that do, however, and they're special cased in fake tensor right now.